### ISSUE TITLE
[Idea] Create "this week in parliament"

### ISSUE THREAD

**Issue by henare** [openaustralia/publicwhip#1126]
Create a page that quickly shows you interesting votes in parliament by way of a simple paragraph. This would need to be manually created by an editor each week. This paragraph could then be customised for every MP and displayed on their page or even **syndicated in local newspapers**. It would looks something like this:

>This week in parliament your MP, Jane Smith, voted for a bill to repeal the carbon tax and voted against a motion in support of equal marriage. They also voted for extending the Northern Territory intervention and against a motion in support of the launch of the righttoknow.org.au website. Jane Smith had an overall vote attendance rate this week of 86% or 12 our of a possible 14 votes.

Of course that paragraph would be peppered with links to allow you to find out more.

**Comment by MicaelaMicaela** [comment]
Good idea! Though I'm not sure that it would be practical (or possible) to do manually in real time - i.e. writing about 'this week in parliament' in the week it actually happened. How about 'last week in parliament'? 

Or perhaps 'what's happening in parliament' would be better so that we're not restricting ourselves to a particular time frame that may be hard to keep up with - i.e. sometimes there'll be something big that we may want to put in this section immediately (e.g. in the same week) and other times there could be delays that make it later.

**Comment by mlandauer** [comment]
I like this but I don't think it's realistic to consider doing. This would bring us much more in the territory of a news publication. Is this something that we could partner with a news organization to do? Until then I'm closing this
